lib/Doctrine/DBAL/Schema/Schema.php

```diff
@@ -73,7 +73,7 @@ class Schema extends AbstractAsset
     protected $_sequences = [];
 
     /**
-     * @var \Doctrine\DBAL\Schema\SchemaConfig
+     * @var \Doctrine\DBAL\Schema\SchemaConfig|bool
```

Comment:
A nullable type would make more sense here.

Reply:
Can't be done in 2.x I guess...

Reply:
Let's ignore it and fix instead of introducing an intermediate incorrect type.

Reply:
Please ignore.